.travis.yml

```diff
@@ -36,6 +36,7 @@ install:
 
 script:
   - if [[ -z "$NO_COVERAGE" ]]; then COVERAGE_OPT="--cov"; else COVERAGE_OPT=""; fi
+  - flake8 .
```

Comment:
In a few projects we've been running `flake8` _after_ `py.test` as failures during tests are a bit more interesting than lint failures so we'd prefer if flake8 didn't block the test run. But it's up to you :)